### PR TITLE
Add documentation for multi attribute login feature

### PR DESCRIPTION
**Pull request by Chathurangap688**
When a user enters a login identifier, IS will map it into the relevant claim according to the regex pattern of the login identifier. Then IS will search the user through userstores by the claim uri and claim value. Finally, if the user enters the correct claim value and that claim was enabled for login by admin, the user can be able to login their account.

https://github.com/wso2-extensions/identity-governance/pull/487
https://github.com/wso2-extensions/identity-local-auth-basicauth/pull/94
https://github.com/wso2-extensions/identity-local-auth-api/pull/31
https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1554
https://github.com/wso2-extensions/identity-outbound-auth-requestpath-basicauth/pull/36
https://github.com/wso2/carbon-identity-framework/pull/3416
https://github.com/wso2/product-is/pull/11471

### CODE REVIEW

File: en/docs/learn/multi-attribute-login.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+## Configuring WSO2 IS for Multi Attribute Login
+
+1. Start the Identity Server and log into the management console using your tenant credentials.
+
+!!! info
+    If you use multiple tenant domains, you need to configure multi attribute login in tenantwise.
+
+2. Click Resident under Identity Providers found in the Main tab and expand the **Account Management** tab.
+
+3. Expand the **Multi Attribute Login** tab and select the **Enable Multi Attribute Login** checkbox.
```
Comment (gomathyk) [inline en/docs/learn/multi-attribute-login.md:17]:
```suggestion
3. Expand **Multi Attribute Login** and select **Enable Multi Attribute Login**.
```

File: en/docs/learn/multi-attribute-login.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+4. Then add clam URIs which allow for multi attribute login in the given text box.
```
Comment (gomathyk) [inline en/docs/learn/multi-attribute-login.md:19]:
```suggestion
4. Then add claim URIs which allow for multi-attribute login in the given text box.
```

File: en/docs/learn/multi-attribute-login.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                <td>^(\+\d{1,2}\s?)?1?\-?\.?\s?\(?\d{3}\)?[\s.-]?\d{3}[\s.-]?\d{4}$</td>
+            </tr>
+            <tr class="odd">
+                <td>http://wso2.org/claims/username</td>
+                <td>^[a-zA-Z0–9._-]{3,}$</td>
+            </tr>
+        </tbody>
+    </table>
+
+You have now successfully set up WSO2 Identity Server for enable multi attribute login.
```
Comment (gomathyk) [inline en/docs/learn/multi-attribute-login.md:55]:
```suggestion
You have now successfully set up WSO2 Identity Server to enable multi-attribute login.
```

File: en/docs/learn/multi-attribute-login.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+3. Expand the **Multi Attribute Login** tab and select the **Enable Multi Attribute Login** checkbox.
+
+4. Then add clam URIs which allow for multi attribute login in the given text box.
+
+   ![adding-claims-for-multi-attribute-login](../assets/img/learn/multi-attribute-login/adding-claims-for-multi-attribute-login.png)
+
+5. Add Regular Expression for Allowed Claims. 
+
+   Once you have configured WSO2 Identity Server for multi attribute login, you need to provide regular expression for multi attribute allowed claims. Some claims already have a default regex, if not you need to provide it under the claim edit window.
+   Select claim from claim list and open the Edit window. Then enter the Regex pattern under Regular Expression field.
```
Comment (gomathyk) [inline en/docs/learn/multi-attribute-login.md:26]: Give instructions to access this in the Management Console.

File: en/docs/learn/multi-attribute-login.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+using any of following flow. 
+
+1. Identifier first authenticator
+2. Basic Authenticator
+3. Request path authenticator
+4. Authentication REST APIs
+5. Oauth Password grant
+6. Password recovery flow
+
+!!! Note
```
Comment (gomathyk) [inline en/docs/learn/multi-attribute-login.md:69]:
```suggestion
!!! Note "What happens if two users use the same value for the same claim?"
```

File: en/docs/learn/multi-attribute-login.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+   What happens if two user use same value for a same claim?
```
Comment (gomathyk) [inline en/docs/learn/multi-attribute-login.md:70]:
```suggestion

```

File: en/docs/learn/multi-attribute-login.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+   If two users use the same value for the same claim, the multi attribute login feature
+   does not support those claims for those users. If you have keep uniqueness of claims,
```
Comment (gomathyk) [inline en/docs/learn/multi-attribute-login.md:73]:
```suggestion
   does not support those claims for those users. Retaining the uniqueness of claim values avoids this conflict.
```

File: en/docs/learn/multi-attribute-login.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+   you can avoid this kind of issues.
```
Comment (gomathyk) [inline en/docs/learn/multi-attribute-login.md:74]:
```suggestion
```

**Review by gomathyk — CHANGES_REQUESTED** [review]

File: en/docs/learn/multi-attribute-login.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
+# Multi Attribute Login
```
Comment (gomathyk) [inline en/docs/learn/multi-attribute-login.md:1]: Refer https://github.com/wso2/docs-is/pull/2553#discussion_r626266442

**Review by gomathyk — CHANGES_REQUESTED** [review]

File: en/docs/learn/multi-attribute-login.md (hunk 4 above)
Comment (Chathurangap688) [reply en/docs/learn/multi-attribute-login.md:29]: fixed: ebaf7a6

File: en/docs/learn/configuring-uniqueness-of-claims.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
+# Configuring Uniqueness of Claims
```
[the remark this answers is not in the document]
Comment (Chathurangap688) [reply en/docs/learn/configuring-uniqueness-of-claims.md:1]: fixed: c0c0d68

File: en/docs/learn/configuring-uniqueness-of-claims.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+                <td>true</td>
+            </tr>
+        </tbody>
+    </table>
+
+   ![additional-claim-properties](../assets/img/learn/multi-attribute-login/additional-claim-properties.png)
+
+6. Finally click the `Update` button to save additional claim property. 
+
+Now you can check and verify the functionality adding users with existing claim value or update user claim value to existing value.
```
[the remark this answers is not in the document]
Comment (Chathurangap688) [reply en/docs/learn/configuring-uniqueness-of-claims.md]: fixed: cc69f72

File: en/docs/learn/configuring-uniqueness-of-claims.md [hunk 12]
```diff
@@ -44,3 +44,5 @@ Follow the steps below to configure WSO2 Identity Server.
 6. Finally, click the `Update` button to save the additional claim property. 
 
 Now you can check and verify the functionality by adding users with existing claim value or update the user claim value of an existing claim.
+If you are successfully configured, you can't create or update users with existing claim value.
```
Comment (gomathyk) [inline en/docs/learn/configuring-uniqueness-of-claims.md:47]: If you have successfully carried out the above-mentioned steps, you will not be able to create or update users with an existing claim value.

File: en/docs/learn/configuring-uniqueness-of-claims.md [hunk 13]
```diff
@@ -44,3 +44,5 @@ Follow the steps below to configure WSO2 Identity Server.
 6. Finally, click the `Update` button to save the additional claim property. 
 
 Now you can check and verify the functionality by adding users with existing claim value or update the user claim value of an existing claim.
+If you are successfully configured, you can't create or update users with existing claim value. 
+It will show an error message saying you can not create or update users with existing claim value.
```
Comment (gomathyk) [inline en/docs/learn/configuring-uniqueness-of-claims.md:48]: Shall we remove this sentence? It looks like a repetition.

File: en/docs/learn/multi-attribute-login.md [hunk 14]
```diff
@@ -7,10 +7,13 @@ how to configure this.
 
 ## Configuring WSO2 IS for Multi Attribute Login
 
-1. Start the Identity Server and log in to the management console using your tenant credentials.
+1. Start the Identity Server and log in to the [Management Console](https://localhost:9443/carbon) using your tenant credentials.
```
Comment (gomathyk) [inline en/docs/learn/multi-attribute-login.md:10]: Shall we use this instead of https://localhost:9443/carbon? 

`https://<IS_HOST>:<PORT>/carbon`

**Review by gomathyk — APPROVED** [review]

File: en/docs/learn/multi-attribute-login.md (hunk 14 above)
Comment (Chathurangap688) [reply en/docs/learn/multi-attribute-login.md]: Fixed: abde67e

File: en/docs/learn/configuring-uniqueness-of-claims.md (hunk 13 above)
Comment (Chathurangap688) [reply en/docs/learn/configuring-uniqueness-of-claims.md]: Removed: abde67e

File: en/docs/learn/configuring-uniqueness-of-claims.md (hunk 12 above)
Comment (Chathurangap688) [reply en/docs/learn/configuring-uniqueness-of-claims.md]: Fixed: abde67e

**Pull request merged**